module-2/assignments/activity1.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<People>
+    <Name>Rorie Demanche</Name>
+    <home>+1 (710) 153-4949</home>
+    <work>+1 (503) 319-3558 </work>
+    <mobile>+1 (182) 748-1667</mobile>
```

Comment:
```suggestion
    <phone type="home">+1 (710) 153-4949</phone>
    <phone type="work">+1 (503) 319-3558 </phone>
    <phone type="mobile">+1 (182) 748-1667</phone>
```

since it is used to store similar data, better to have same element name but with attribute to specify details about it